tardis/plasma/base.py

```diff
@@ -26,7 +27,8 @@ def __init__(self, plasma_properties, property_kwargs=None, **kwargs):
             plasma_properties, property_kwargs, **kwargs
         )
         self._build_graph()
-        #        self.write_to_tex('Plasma_Graph')
+        self.write_to_dot("Plasma_Graph.dot")
```

Comment:
why?

Reply:
I assumed that TARDIS should automatically output the .dot file in addition to the .tex file as well for users who want to use the .dot file.

Reply:
This will be removed in the next commit